packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponent.tsx

```diff
@@ -122,7 +128,7 @@ export const ChatMyMessageComponent = (props: ChatMyMessageComponentProps): JSX.
 
   if (isEditing && message.messageType === 'chat') {
     return (
-      <ChatMessageComponentAsEditBox
+      <ChatMessageComponentAsEditBoxSelector
```

Comment:
I feel in our repo, selector is used to for the handler that connects the adapter and the UI part of the code. Maybe we should rename this component to reduce the confusion.